ssl.pl

```diff
@@ -78,6 +78,7 @@
 		       certificate_file(atom),
 		       key_file(atom),
 		       password(any),
+		       cipher_list(any),
```

Comment:
If this is an atom, the argument must be atom.   Shouldn't we make this a list though?   Joining the list is way easier on the Prolog side, so that would imply adding a Prolog wrapper.  Not sure this is worth the trouble?  What do you think?